solidity/test/integration/Deployment.test.ts

```diff
@@ -236,4 +239,40 @@ describe("Deployment", () => {
       )
     })
   })
+
+  describe("AcreMultiAssetVault", () => {
+    testUpgradeableInitialization(
+      "AcreMultiAssetVault",
+      ZeroAddress,
+      ZeroAddress,
+      [ZeroAddress],
+    )
+
+    it("should set mezoPortal", async () => {
+      expect(await multiAssetVault.mezoPortal()).to.be.equal(
+        expectedMainnetAddresses.mezoPortal,
+      )
+    })
+
+    it("should set supportedAssets", async () => {
+      const expectedSupportedAssets = [
+        expectedMainnetAddresses.solvBtc,
+        expectedMainnetAddresses.solvBtcBbn,
+      ]
+
+      for (let i = 0; i < expectedSupportedAssets.length; i += 1) {
+        expect(
+          // eslint-disable-next-line no-await-in-loop
+          await multiAssetVault.supportedAssets(expectedSupportedAssets[i]),
+          `asset ${i}/${expectedSupportedAssets.length} is not supported: ${expectedSupportedAssets[i]}`,
+        ).to.be.true
+      }
+    })
+
+    it("should set owner", async () => {
+      expect(await multiAssetVault.owner()).to.be.equal(
+        expectedMainnetAddresses.governance,
+      )
+    })
```

Comment:
Could be worth finding the ProxyAdmin and asserting its owner as well.

Reply:
f0f1a41b6ba8938f34c633bec323ffb703e9bf54